AIMSICD/src/main/java/com/SecUpwN/AIMSICD/service/CellTracker.java

```diff
@@ -965,7 +962,7 @@ public void cancelNotification() {
      *      See:  https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/wiki/Status-Icons
      *      and:  https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/issues/11#issuecomment-44670204
      *
-     *      Change names from "IDLE,NORMAL,MEDIUM,ALARM" to:"GRAY,GREEN,YELLOW,ORANGE,RED,BLACK",
+     *      Change names from "IDLE,OK,MEDIUM,ALARM" to:"GRAY,GREEN,YELLOW,ORANGE,RED,BLACK",
```

Comment:
Should be status `HIGH`. Should I remove these irritating comments? We have multiple.